AddOns/BasicFinancials/app/src/codeunit/AppAreaMgmt.Codeunit.al

```diff
@@ -0,0 +1,28 @@
+codeunit 20600 "App Area Mgmt BF"
```

Comment:
The name of the file should also reflect the name of the codeunit, so instead of AppAreaMgmt.Codeunit.al it must be AppAreaMgmtBF.Codeunit.al. Otherwise, we might have files with the same name.

Reply:
Applies for all other files